cfme/common/vm.py

```diff
@@ -455,8 +455,8 @@ def instantiate(self, name, provider, template_name=None):
         else:
             return provider.vm_class.from_collection(self, name, provider, template_name)
 
-    def create(self, vm_name, provider, form_values=None, cancel=False,
-               check_existing=False, find_in_cfme=False, wait=True):
+    def create(self, vm_name, provider, form_values=None, cancel=False, check_existing=False,
+               find_in_cfme=False, wait=True, extra_msg='', approve=False):
```

Comment:
**REQUIRED:** I'm not sure I like ```extra_msg```, what about giving the ability to totally override the ```request_description```?

Reply:
@psav I think we need that value to be present, whenever we request for vm more than 1 we need an extra message as '###' to be part of request_description.